src/Worlds.js

```diff
@@ -90,3 +137,22 @@ export function isWorldName(str: ?string): boolean {
 export function getWorldProperties(world: WorldName): WorldProperties {
     return worlds[world];
 }
+
+export function getBackgroundInfo(world: WorldName, yPos: number, xPos: number): ?string {
+    const worldBackgroundInfo = worlds[world].backgroundInfo;
+    if (worldBackgroundInfo) {
+        return worldBackgroundInfo[yPos][xPos];
```

Comment:
I can actually get the environment to crash, which is a sign to me that there are cases in which `worldBackgroundInfo` is set but not all rows/columns are.  Try moving one square from H7 to I7 in the Space world.

Reply:
Looks like anything higher than H causes a crash.

Reply:
Crash was caused by calling the method in CharacterAriaLive.js and Scene.js with wrong parameter order